packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -481,11 +481,9 @@ describe("MafsGraph", () => {
             />,
         );
 
-        // The middle coords are actually the first point because we want
-        // the vertex to show up first in the point tab order.
-        expectLabelInDoc("Point 1 at 0 comma 0");
-        expectLabelInDoc("Point 2 at -1 comma 1");
-        expectLabelInDoc("Point 3 at 1 comma 1");
+        expectLabelInDoc("Point 1, terminal side at -1 comma 1");
```

Comment:
Can we add some tests here for the aria-label for when it reads out the vertex angle? :)